WcaOnRails/app/models/country.rb

```diff
@@ -56,7 +56,9 @@ def self.localized_sort_by!(wca_locale, array, &block)
       'pt-BR': 'pt',
       'zh-CN': 'zh',
       'zh-TW': 'zh-Hant',
+      'sl': 'hr',
```

Comment:
This looks great, but could we make an issue for this and add a link to it here?